### PR TITLE
Make user configuration for unifi-os clearer.

### PR DESCRIPTION
**Pull request by florisvdk**
## Proposed change

Make the configuration for unifi-os clearer. (udmp and soon udm, ckg2)

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by fabaff — CHANGES_REQUESTED** [review]
Ubiquiti is using "UniFi OS" and not "Unifi-os".

Could you please change it?

File: source/_integrations/unifi.markdown [hunk 1]
```diff
@@ -37,6 +37,10 @@ All configuration options are offered from the front end. Enter what UniFi integ
 
 The UniFi controller allows you to create multiple users on it besides the main administrator. If all you want to use is the device tracker then it is recommended that you create a limited user that has `read-only` permissions for the UniFi device tracker. If you want blocking of network access or POE control as well you would need to have 'admin' permissions.
 
+### Unifi-os
+
+For Unifi-os a local only user needs to be created. A user which uses the Ubiquiti cloud will not work. You can do this in the manage users section on the Unifi-os dashboard. Make sure to give it the right permissions for the functions you want to use.
```
Comment (frenck) [inline source/_integrations/unifi.markdown]:
```suggestion
For Unifi-os a local-only user needs to be created. A user who uses the Ubiquiti cloud will not work. You can do this in the manage users section on the Unifi-os dashboard. Make sure to give it the right permissions for the functions you want to use. 
```

File: source/_integrations/unifi.markdown [hunk 2]
```diff
@@ -37,6 +37,10 @@ All configuration options are offered from the front end. Enter what UniFi integ
 
 The UniFi controller allows you to create multiple users on it besides the main administrator. If all you want to use is the device tracker then it is recommended that you create a limited user that has `read-only` permissions for the UniFi device tracker. If you want blocking of network access or POE control as well you would need to have 'admin' permissions.
 
+### Unifi-os
```
Comment (frenck) [inline source/_integrations/unifi.markdown]:
```suggestion
### UniFi OS
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @florisvdk! 👍

**Pull request merged**